shell/client/shell.html

```diff
@@ -1078,6 +1070,51 @@ <h4 class="share-with-others">Share with others</h4>
       <p>To get support for LDAP, SAML, and organization management, enable
         <a href="{{featuresPath}}">Sandstorm for Work</a>.</p>
     {{/if}}
+    <div class="legacy-email-config">
+      <h4>SMTP Configuration: <button id="admin-settings-send-toggle">Test</button></h4>
```

Comment:
Nitpick: It feels weird to me that the Sandstorm for Work features would be inserted between non-SFW feature sections rather than appended to the end. Maybe this should stay before LDAP login? I understand the desire to group identity service checkboxes together, but LDAP is already separated from the others by an `<h4>` so I'm not sure there can be said to be any such grouping currently.